scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -2,7 +2,13 @@
 This is an project created to use Maven-published Scala/Java package with two Java examples.
 ## Setup
 Please copy the downloaded MXNet Java package jar file under the `java-demo` folder.
+You can download the package from [here](https://repository.apache.org/#nexus-search;quick~mxnet).
+Please download `1.3.1-SNAPSHOT` or above version to test to run this demo.
```

Comment:
will 1.3.1-SNAPSHOT work?

Reply:
Yes, because it take the up-to-date master